README.md

```diff
@@ -200,19 +200,14 @@ ipsec auto --add <connectionname>
 
 Any errors that may have occurred during the process of loading and starting the connection are in the logs, which can be found in `/var/log/pluto.log` in RHEL 8, or by issuing the command `journalctl -u ipsec` in RHEL 7. Since these logs can be verbose and contain old entries, it is generally recommended to try to manually add the connection to obtain log messages from the standard output instead.
 
-## Firewall
+## Firewall and Selinux
 
-The firewall must be configured to allow traffic on 500 and 4500/UDP ports for the IKE, ESP, and AH protocols. In the future, the firewall system role may be used for this configuration, however in the meantime these settings will need to be manually configured.
+The firewall must be configured to allow traffic on 500 and 4500/UDP ports for the IKE, ESP, and AH protocols.
```

Comment:
Do we also want to allow 4500/TCP for IKE over TCP https://libreswan.org/wiki/RFC_8229_-_TCP_support_for_IKEv2_and_ESP?

Reply:
Actually, we do as we enables `ipsec firewall service` which includes it.
```
<service>
  <short>IPsec</short>
  <description>Internet Protocol Security (IPsec) is the standarized IETF VPN architecture defined in RFC 4301. IPsec is negotiated using the IKEv1 (RFC 2409) or IKEv2 (RFC 7296) protocol, which in itself uses encryption and authentication. IPsec provides Internet Protocol (IP) packet encryption and authentication. Both IKE and IPsec can be encapsulated in UDP (RFC 3948) or TCP (RFC 8229 to make it easier to traverse NAT. Enabling this service will enable IKE, IPsec and their encapsulation protocols and ports. Note that IKE and IPsec can also be configured to use non-default ports, but this is not common practise.</description>
  <port protocol="ah" port=""/>
  <port protocol="esp" port=""/>
  <port protocol="udp" port="500"/>
  <port protocol="udp" port="4500"/>
  <port protocol="tcp" port="4500"/>
</service>
```
I'm going to update the README. Thanks.